net/fetcher.go

```diff
@@ -22,6 +30,212 @@ type Fetcher interface {
 	FetchTipSets(ctx context.Context, tsKey types.TipSetKey, from peer.ID, done func(ts types.TipSet) (bool, error)) ([]types.TipSet, error)
 }
 
+// GraphSyncFetcher is used to fetch data over the network.  It is implemented with
+// a persistent bitswap session on a networked blockservice.
+type GraphSyncFetcher struct {
```

Comment:
Please move this to a new file `graphsync.go`. We shouldn't have put the bitswap fetcher in here. Also the comment is wrong (I hope).

Reply:
for sure